### PR TITLE
Add/restore go-to-top button examples

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/920
[Problem] No go-to-top button examples
[Solution] Add/restore examples.

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/navigationelements/index.html [hunk 1]
```diff
@@ -50,6 +50,11 @@ <h2 class="ui-title">
 						Assist Panel
 					</a>
 				</li>
+				<li>
+					<a href="gototopbutton/index.html">
```
Comment (grzegorz-czajkowski) [inline examples/wearable/UIComponents/contents/navigationelements/index.html]: go-to-top-button ?

Comment (lslachciak) [reply examples/wearable/UIComponents/contents/navigationelements/index.html]: @grzegorz-czajkowski good question. I was thinking about it
Not sure if kebap-case style refers also to directories. We need to find consensus

Comment (lslachciak) [reply examples/wearable/UIComponents/contents/navigationelements/index.html]: ok, decided to change this

**Review by grzegorz-czajkowski — APPROVED** [review]
Looks fine.

**Pull request merged**